### ISSUE TITLE
Question 6.3 still remains unanswered

### ISSUE THREAD

**Issue by SuyashD95** [SuyashD95/cplusplus-prog-exercises#59]

**Comment by SuyashD95** [comment]
**Update 1**:

Things that I have achieved in this update:

1. Created a constructor which allocates the desired memory to the members using 'new' operator.
2. Populated the members with data.

**NOTE:**
Even though I have uploaded this update, there are some areas/ concerns with this code that I want to draw your attention...

- First, is the use of C-style arrays i.e, char arrays or char*... In modern C++, especially if you are using a C++ compiler based on C++11 or C++14 standard (or a later standard), it is _**highly recommended**_ to use the _std::string_ container to deal with string or multi-character based data...

- Second, even though, I have used 'new' (since, it was mentioned in the question), I will also need to 
  define a destructor to deallocate the memory that has been allocated to the members... But, again, this 
  complicates the situation since the responsibility of dealing with releasing the memory and preventing 
  situations like memory leaks fall on our shoulders...
  
  Hence, **_only use 'new' when it is absolutely necessary_** (or, in situations, when the question has 
  explicitly mentioned to use it..) The program is working fine and it is terminating properly (hence, I feel 
  that the code is freeing the memory properly, but again, be careful while using new and delete).
